roles/console/files/help/Config.rst

```diff
@@ -1,171 +1,177 @@
 
-* **Samba** - Provides Network File Sharing.
+### Firewall
 
-    There is some overlap between these three.  **Elgg** allows blogging and other forms of social media.  Students and Teachers can use it to collaborate on projects or for journaling.  **Nextcloud** is great for sharing media. It has apps for phones and tablets that make it easy to drop photos and other materials onto the server for sharing. **Samba** gives you the ability to share directories on the server that can be accessed by Teachers and Students as if they were local to their laptops.
+You will likely not change these.
 
-* **CUPS** - Provides support for **Printers** either directly attached to the server or on the network.
+* **Inbound Ports**
 
-### For Monitoring and Administration
+Determines which if any ports will be open to the WAN
 
-The options below are intended for administrators and people who may help with or support the installation of this Server. It is best to consult with someone who set up the server to decide which of these to turn on.
+* **Outbound Traffic**
 
-* **SchoolTool** - A School Administration System.
+Allows netowrk traffic to be routed from LAN to WAN
 
-* **XO Visualization** - Graphs of Student Usage Statistics.
-* Title to Appear on XO Visualization Charts
+Internet Caching/Filtering
+--------------------------
 
-* **Collect Statistics** - on the use of Sugar Activities.
+**Please Note**: These parameters only take effect when you have selected the Gateway Role on the previous screen and have the necessary Network Adapters.
 
-* **Monit** - Watches critical applications and restarts them if they fail.
+* Enable local Web Page storage for later fast access (Squid cache)
 
-* **Munin** - Collects and graphs system-level statistics.
+* Restrict Web Page Access to a Specific List of Sites (Permitted URLs)
 
-* **vnStat** - Gathers and displays networking statistics.
+* Block all HTTPS Access to Web Page
 
-* **AWStats** - Graphs statistics on web server usage.
+When the Server acts as a **Gateway** between users and the Internet it can provide two main types of services.  The first is that it can make the connection to the Internet more efficient by caching or storing responses on the server so that the next request does not need to go to the Internet. You will usually want this.
 
-* **phpMyadmin** - Allows maintenance of MySQL databases.
+In addition, it can filter the sites that students are permitted to access in three ways.  The first is to only allow pages in the **Permitted URLs** list to be accessed; see below.  The second is that all sites accessed with https security can be blocked to prevent users from by-passing the previous filter.
 
-* **OpenVPN** - Allows a secure connection between servers over the Internet for remote maintenance. You can access via a terminal or a web browser.
+The last filter is based not on URLs, but on **Words in the Content**.
+
+Use your local policy to decide which of these filters to enable.
```

Comment:
Should these 2 lines be removed or revised?

And Line 164's "it can filter the sites that students are permitted to access in three ways" be revised accordingly?